aces/Loris
# [issue_tracker] Notifications

## Brief summary of changes

- Fixed issue where the assignee dropdown did not show the current assignee
- Emails users who were added as watching
- When a comment is added, it adds it to the email to everyone watching

#### Testing instructions (if applicable)

1. Open the issue tracker and add a user with your email address as the watcher to an issue
2. Confirm that you received an email about the issue
3. Try changing various fields and confirm an email is sent for each with some context.

#### Link(s) to related issue(s)

CCNA Overrides:
https://github.com/aces/CCNA/pull/3889
https://github.com/aces/CCNA/pull/4618
